routes/teams.js

```diff
@@ -40,4 +43,115 @@ router.get('/team', function(req, res) {
     });
 });
 
+
+// when players endpoint is hit, call the API/DB using that team_id
+router.get('/players', function(req, res) {
+  team_id = req.query["team_id"];
+  console.log("team id is: "+team_id);
+  players = fetchPlayers(team_id);
+  // I think this res.json() fires too quickly. But I don't understand callbacks soooo
+  res.json(players);
+});
+
+
+
+
+
+  
+var fetchPlayers = function(team_id){ 
+      db_players.collection('players').find({team_id : team_id}).toArray(function (err, items){
+        secondFunction(items, team_id);
```

Comment:
It's not that simple. If I just assign it to a variable and return it, the return runs before the db call is complete so I return an empty item.  But I can't figure out how to make it wait for the data to return first.